apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/ValidatorApp.scala

```diff
@@ -199,6 +206,14 @@ class ValidatorApp(
       initialSynchronizerTime <-
         withParticipantAdminConnection { participantAdminConnection =>
           for {
+            _ <- Future.unit
```

Comment:
this is a bit confusing, I would just move the internalStore definition outside the for in this case